bofire/strategies/predictives/botorch.py

```diff
@@ -332,91 +360,202 @@ def _optimize_acqf_continuous(
+        """
+        reference_experiment = self.experiments.loc[  # type: ignore
+            self.trust_region_config.X_center_idx  # type: ignore
+        ]
+        # instantiate a random strategy and ask it for the new experiments.
+        inputs = self.domain.inputs
+        outputs = self.domain.outputs
+        constraints = self.domain.constraints
+
+        # This is a bit of a hack to get the local sampler to work, ideally
+        # there would be a more elegant solution.
+        local_inputs = Inputs(
+            features=[
+                ContinuousInput(
+                    key=inp.key,
+                    bounds=(
+                        max(
+                            reference_experiment[inp.key]
+                            - self.trust_region_config.length / 2,  # type: ignore
+                            inp.lower_bound,  # type: ignore
+                        ),
+                        min(
+                            reference_experiment[inp.key]
+                            + self.trust_region_config.length / 2,  # type: ignore
+                            inp.upper_bound,  # type: ignore
+                        ),
+                    ),
+                )
+                for inp in inputs.get(ContinuousInput)
+            ]
+            + list(inputs.get(excludes=ContinuousInput))  # type: ignore
+        )
+        local_domain = Domain(
+            inputs=local_inputs,
+            outputs=outputs,
+            constraints=constraints,
+        )
+        sampler = strategies.map(
+            data_model=RandomStrategyDataModel(domain=local_domain)
+        )
+        # we have to ask for candidate_count to not break the validation elsewhere
+        # in bofire. This is inefficient and breaks the low discrepancy sampling
+        # benefits of using a single sobol sequence.
+        candidates = sampler.ask(candidate_count)
+        for key in self.domain.outputs.get_keys():
+            candidates[f"{key}_pred"] = 0
+            candidates[f"{key}_sd"] = 0
+            candidates[f"{key}_des"] = 1
+        return candidates
+
+    def _ask(self, candidate_count: int) -> pd.DataFrame:  # type: ignore
+        """Ask the strategy for new experiments.
 
+        Args:
+            candidate_count (int): The batch size of new experiments
+                requested from the strategy. Defaults to 1.
         """
         assert candidate_count > 0, "candidate_count has to be larger than zero."
         if self.experiments is None:
             raise ValueError("No experiments have been provided yet.")
 
+        if (
+            self.trust_region_config is not None
+            and not self.trust_region_config.has_sufficient_experiments(
+                self.experiments, self.domain
+            )
+        ):
+            # fill the trust region with random samples if we're using a trust
+            # region approach and don't have enough experiments to fit a
+            # reasonable GP surrogate
```

Comment:
I think this linting error is caused by starting the comment with the word `region`: https://github.com/microsoft/pylance-release/issues/3765. Just format it differently.